lib/generators/rolemodel/css/icons/icons_generator.rb

```diff
@@ -0,0 +1,20 @@
+module Rolemodel
+  module Css
+    class IconsGenerator < Rails::Generators::Base
+      source_root File.expand_path('templates', __dir__)
+
+      def add_material_icons
+        run 'yarn add material-icons'
+      end
+
+      def add_view_helper
+        template 'app/helpers/icon_helper.rb'
```

Comment:
I believe you could use `copy_file` here too, since you aren't actually doing anything with the interpolating the template file itself.